components/SSI/DIDxWallet/Wallet/Allowances/index.tsx

```diff
@@ -65,18 +102,17 @@ function Component() {
         const spender_ = {
           vname: "spender",
           type: "ByStr20",
-          value: "0x54eabb9766259dac5a57ae4f2aa48b2a0208177c", //@todo-i add input address for spender
+          value: spender, //@todo-i-checked add input address for spender
         };
         params.push(spender_);
         const amount_ = {
           vname: "amount",
           type: "Uint128",
-          value: amount, //todo-i amount times the decimals
+          value: amount, //todo-i-? amount times the decimals
```

Comment:
this means that the value we have to send to the blockchain is the transaction amount, e.g. 10 TYRON with its decimals is 10e12